src/construction/construction_helpers.py

```diff
@@ -63,20 +63,26 @@ def prepare_forms_gb(
     if "construction_type" in construction_df.columns:
         # Prepare the short to long form constructions, if any (N/A to NI)
         if "short_to_long" in construction_df.construction_type.unique():
-            snapshot_df = prepare_short_to_long(snapshot_df, construction_df)
+            snapshot_df, unique_references = prepare_short_to_long(
+                snapshot_df, construction_df
+            )
     # Create period_year column (NI already has it)
     snapshot_df = create_period_year(snapshot_df)
     construction_df = create_period_year(construction_df)
     # Set instance=1 so longforms with status 'Form sent out' match correctly
     form_sent_condition = (snapshot_df.formtype == "0001") & (
         snapshot_df.status == "Form sent out"
     )
-    snapshot_df.loc[form_sent_condition, "instance"] = 1
```

Comment:
this is nice. Maybe a comment to explain why we're excluding the ones that have been fully constructed as long forms with possibly multiple instances?